djstripe/models/payment_methods.py

```diff
@@ -181,7 +230,7 @@ class BankAccount(LegacySourceMixin, StripeModel):
         on_delete=models.PROTECT,
         null=True,
         blank=True,
-        related_name="bank_account",
+        related_name="bank_accounts",
```

Comment:
This is backwards incompatible, can you add a .bank_account alias property?

Reply:
Ok sure. I'll do that.